Vagrantfile

```diff
@@ -156,6 +156,22 @@ set -o pipefail
 # something that we don't want to happen when running provision in a
 # development environment not using Vagrant.
 
+# Only show /etc/motd and nothing else on vagrant ssh
+sudo rm /etc/update-motd.d/*
+
+# Set MOTD message to our custom message
+sudo bash -c 'cat << EndOfMessage >> /etc/motd
```

Comment:
I think we want this to go into `update-motd.d` or something -- otherwise if you provision multiple times, you'll end up with multiple copies of this in the motd.

Reply:
The line "sudo rm -f /etc/motd" will delete the motd file if it exists. This is better than update-motd.d because update-motd.d is for scripts and adding a message to /etc/motd is the standard way to set a motd.

Reply:
The line "sudo rm -f /etc/motd" removes the motd file if it exists. This is better since adding text to /etc/motd is the standard way to set a motd whereas update-motd.d is intended for scripts.